tests/clipboard/clipboard_plugin.test.ts

```diff
@@ -2011,6 +2011,28 @@ describe("clipboard", () => {
     paste(model, "G4");
     expect(getCell(model, "G4")!.content).toBe("=PIVOT(1)");
   });
+
+  test("fixed pivot formulas are copied like standard cells", () => {
+    const grid = {
+      A1: "Customer",
+      B1: "Price",
+      C1: "1",
+      A2: "Alice",
+      B2: "10",
+      C2: '=PIVOT.VALUE(C1,"Price","Customer","Bob")',
+      A3: "Bob",
+      B3: "30",
+    };
+    const model = createModelFromGrid(grid);
+    addPivot(model, "A1:B3", {
+      columns: [],
+      rows: [{ name: "Customer" }],
+      measures: [{ name: "Price", aggregator: "sum" }],
+    });
```

Comment:
Nitpick: most of this setup is useless if the goald is jsut to test the copy/paste of a singel formula right ?

Reply:
yes and no, I think it's best to have a working pivot (properly defined, loaded initialised etc) to mimic a normal behaviour even if technically speaking it's not required. But I missed a bloody prettier ignore as well...